source/standards/managing-sensitive-information.html.md.erb

```diff
@@ -0,0 +1,399 @@
+settings:
+
+- **Public** repositories are visible to the world and are primarily used for open source content.
+- **Internal** repositories are read-accessible to all full members of your enterprise account (GDS, CDDO, CO:Di.AI and other parts of Cabinet Office but not to outside collaborators).
+- **Private** repositories are only visible to individuals or teams who have been given explicit access, including through enterprise or organization base permissions.
+
+See [GitHub resource for more information](https://resources.github.com/learn/pathways/administration-governance/essentials/manage-your-repository-visibility-rules-and-settings/).
+
+## Scope
+
+Teams should make use of Handling Instructions to contextualise and aid decision making around who should have read access to code.
+
+Write and admin access to repositories should be more tightly controlled.
+Good management of GitHub access controls, such as using infrastructure as code, is a separate concern.
+
+Where targeted controls are required to limit who can review and merge changes, [CODEOWNERS][code-owners] should be used as the mechanism.
+
+## GitHub repositories to include handling instructions
+
+New GitHub repositories should follow the requirements of this section.
+Teams should aim for existing repositories to become compliant over time (see [remediation](#costs-benefits-and-remediation)).
+Lead technologists may enforce this as policy for their area of responsibility, in which case they must reference this page from their team documentation.
+
+GitHub repository visibility must be:
+
+- aligned with the Service Standard, that is, public by default
+- aligned with the Government Security Classifications Policy, by recording handling instructions that reflect any other choice made
+
+Repository owners should use the following process:
+
+1. Presume a default classification of `OFFICIAL - FOR PUBLIC RELEASE` - public visibility - for all source code generated by the programme, with no handling instructions recorded in this case.
+
+2. Use the Service Manual’s [making source code open and reusable][service-manual-open] advice to decide to restrict visibility, where there are concerns that require deviation from the Service Standard.
+
+In the second case, they must record handling instructions in a file committed into version control of every non public repository, where:
+
+- Internal is mapped to `OFFICIAL-SENSITIVE` with the handling instruction `HMG USE ONLY`: there is an assumption of broad visibility within the GitHub enterprise, and ability to add outside collaborators across Government.
+- Private is mapped to `OFFICIAL-SENSITIVE` with the handling instruction `[INSERT ORGANISATION(S) NAME] USE ONLY` or `RECIPIENTS ONLY` as appropriate.
+
+Where GDS engages contractors, either as Managed Service Providers (MSP) or Contingent Labour (CL), the Programme Office should establish at the point of contract how they are to be treated with respect to information handling.
```

Comment:
is programme office still a commonly used term in GDS? or is this a specific team?